addOns/fuzz/src/main/javahelp/org/zaproxy/zap/extension/fuzz/resources/help/contents/options.html

```diff
@@ -3,26 +3,43 @@
 <HEAD>
 <META HTTP-EQUIV="Content-Type" CONTENT="text/html; charset=utf-8">
 <TITLE>
-Options Fuzz screen
+Options Fuzz Screen
 </TITLE>
 </HEAD>
 <BODY>
-<H1>Options Fuzzer screen</H1>
+<H1>Options Fuzzer Screen</H1>
 <p>
 This screen allows you to configure the <a href="concepts.html">fuzzing</a> options:
 
-<H3>Default category</H3>
-The category that will be initially selected shown when the <a href="dialogue.html">Fuzz dialog</a> is displayed. 
+<H3>Default Category</H3>
+The category that will be initially selected when the <a href="dialogue.html">Fuzz dialog</a> is displayed.
 
-<H3>Concurrent scanning threads per host</H3>
-The number of threads the scanner will use per host.<br>
-Increasing the number of threads will speed up the scan but may put extra strain on the computer ZAP is running on and the target host.
-
-<H3>Add custom Fuzz file</H3>
+<H3>Add Custom Fuzz File</H3>
 Allows you to add your own files to be used when fuzzing.<br>
-These should be text files with one attack per line.<br>
+These should be text files with one payload per line.<br>
 Files are added to the 'fuzzers' directory underneath the ZAP home directory.
 
+<H3>Finished Fuzzers in UI</H3>
+It defines the numbers of fuzzers (that have completed their execution)
+visible in the fuzzer tab.
+
+<H3>Retries on IO Error</H3>
+The number of retries when an input/output error occurs sending a request to the target.
+
+<H3>Max. Errors Allowed</H3>
+If the number of errors exceed this limit, fuzzer will stop its execution.
+
+<H3>Payload Replacement Strategy</H3>
+Rules defined to control the order that multiple payload lists are iterated.
```

Comment:
I think this is fine for now. When the options are fixed this can be revised. (That's my 2 cents anyway.)